lib/model/QubitInformationObject.php

```diff
@@ -1168,6 +1168,22 @@ public function getDigitalObject()
   }
 
   /**
+   * Get the digital object's checksum value
+   *
+   * @return string  digital object checksum or null
+   */
+  public function getDigitalObjectChecksum()
+  {
+    // Set digital object URL
+    $digitalObjects = $this->getDigitalObjects();
```

Comment:
I'd do `if (null !== $do = $this->getDigitalObject())
{
  return $do->getChecksum();
}

return null;`